testinfra/utils/ansible_runner.py

```diff
@@ -383,9 +383,8 @@ def run_module(
                     "msg": "Skipped. You might want to try check=False",
                 }
             if not files:
-                raise RuntimeError(
-                    "Error while running {}: {}".format(" ".join(cmd), out)
-                )
+                cmd = local.backend.get_command(cmd, *args)
+                raise RuntimeError('Error while running "{}": {}'.format(cmd, out))
```

Comment:
`cmd`is same as `out.cmd` and already displayed in `str(out)` no ?

So I suggest:

```python
raise RuntimeError(f"{out}")
```

Reply:
Changed in 5a54e25